.gitignore

```diff
@@ -1,10 +1,10 @@
+# Environment
+.env
+.env*.local
+
+# Generated Files
 out/
 .next
 coverage/
 flow-coverage/
 node_modules/
-.idea/
```

Comment:
I would keep DS_Store we can argue about .idea/ and .vscode/ but it's ok since we are not doing any settings config nor launch specifications here.

Reply:
I would not, because when I use my Comodore 64, I do not add `.G64` files ignores for that file system.  Educate people to keep their personal choice artifacts out of projects and no one has to maintain `DS_Store` in gitignore other than their own personal ones if they choose to use closed operating systems.

Reply:
It is also why I am OK with https://editorconfig.org because it is addressing editors in general, rather than a specific editor choice.  We can say, as an organization or even as a project, "Hey, there are our opinions for files.".

Reply:
got it, approved